Tests/WeaverTests/AssemblyToProcess/RemappedClass.cs

```diff
@@ -28,5 +28,6 @@ namespace AssemblyToProcess
     [MapTo("RemappedTable")]
```

Comment:
are we dropping `MapTo`? If that is the case then we could  drop this entire test